query-node/schema.graphql

```diff
@@ -117,38 +117,48 @@ union DataObjectOwner = DataObjectOwnerMember
 
 "Asset owned by a member"
 type DataObjectOwnerMember @variant {
+  # use `BigInt` instead of `Membership` before variant relations are featured in Hydra
+  #"Member identifier"
+  #memberId: Membership!
   "Member identifier"
-  memberId: Membership!
+  member: BigInt!
 
   "Variant needs to have at least one property. This value is not used."
-  dummy: Int!
+  dummy: Int
 }
 
 "Asset owned by a channel"
 type DataObjectOwnerChannel @variant {
+  # use `BigInt` instead of `Channel` before variant relations are featured in Hydra
+  #"Channel identifier"
+  #channel: Channel!
   "Channel identifier"
-  channel: Channel!
+  channel: BigInt!
 
   "Variant needs to have at least one property. This value is not used."
-  dummy: Int!
+  dummy: Int
 }
 
 "Asset owned by a DAO"
 type DataObjectOwnerDao @variant {
   "DAO identifier"
-  daoId: BigInt!
+  dao: BigInt!
 }
 
 "Asset owned by the Council"
 type DataObjectOwnerCouncil @variant {
   "Variant needs to have at least one property. This value is not used."
-  dummy: Int!
+  dummy: Int
 }
 
 "Asset owned by a WorkingGroup"
 type DataObjectOwnerWorkingGroup @variant {
-  "Working group identifier"
-  workingGroupId: BigInt!
+  # TODO: working group doesn't seem to have id but rather `isStorage`, `isContent`, ... shoudl we care about that?
+  #"Working group identifier"
+  #workingGroup: BigInt!
```

Comment:
Should we somehow handle working groups, or simply stating that "some" working group is owning the asset?

Reply:
You are absolutely right. These "isStorage", etc. properties in types confused me and I forgot it's enum. Fixed